src/test/dynamicResources/awsResourceManager.test.ts

```diff
@@ -192,7 +192,7 @@ describe('ResourceManager', function () {
         const expectedSchemaLocation = path.join(tempFolder, 'sometype.schema.json')
         assert.ok(existsSync(expectedSchemaLocation))
         assert.strictEqual(mapping.type, 'json')
-        assert.strictEqual(mapping.path.toLowerCase(), editor.document.uri.fsPath.toLowerCase())
+        assert.strictEqual(mapping.uri.fsPath.toLowerCase(), editor.document.uri.fsPath.toLowerCase())
```

Comment:
- while we're here would you mind changing this to use `pathUtils.areEqual()`
- same for the other case below